src/server.c

```diff
@@ -3531,6 +3581,8 @@ void call(client *c, int flags) {
         char *latency_event = (real_cmd->flags & CMD_FAST) ?
                                "fast-command" : "command";
         latencyAddSampleIfNeeded(latency_event,duration/1000);
+        if (server.execution_nesting == 1)
+            durationAddSample(EL_DURATION_TYPE_CMD, duration);
```

Comment:
```suggestion
        if (server.execution_nesting == 0)
            durationAddSample(EL_DURATION_TYPE_CMD, duration);
```
i now realize that here we're already after `exitExecutionUnit()`.

Reply:
that's an indication that we don't have a test for it.
please see if you add some trivial test.

Reply:
I wonder how we can test this... We can send `MULTI` `DEBUG SLEEP 1` `EXEC` and then check how much time is added to the metric. But I don't feel it is necessary...

Reply:
with the bug above, nothing will get logged when a plain command is executed, so that's easy to test (just check that we have some value in that metric).
i don't have a strong objection to that DEBUG SLEEP test, but i'm also ok with just having a trivial test that checks the metric isn't empty.